k8s/configMap-sjc.yaml

```diff
@@ -30,9 +30,6 @@ data:
     #    UrlRegexp: ^/info
         FrontendRegexp: services-k8s\.wikia-dev\.(us|pl)
         Sampling: 1.0
-      - Id: geoip-external
-        FrontendRegexp: services\.wikia\.com\/geoip
-        Sampling: 1.0
-      - Id: whoami-external
-        FrontendRegexp: services\.wikia\.com\/whoami
-        Sampling: 1.0
\ No newline at end of file
+      - Id: services-all
+        FrontendRegexp: ^[^\.]+\.k8s\.wikia\.net
```

Comment:
This is duplicating `generic-internal`. This way we have duplicated series with exactly the same data.